pkg/levee/levee.go

```diff
@@ -0,0 +1,7 @@
+// Package levee exports the levee Analyzer.
+package levee
+
+import internal "github.com/google/go-flow-levee/internal/pkg/levee"
+
+// Analyzer reports attempts to source data to sinks.
```

Comment:
Let's avoid using `source` as a verb, so as to avoid the heavy usage of it as a noun throughout the codebase.  Consider instead:
```golang
/// Analyzer reports instances of source data reaching a sink.
```